src/screens/DomainScreen/elements/RenderItem.js

```diff
@@ -237,84 +213,84 @@ const RenderItem = ({
-  contentDetail: { flexDirection: 'row', alignItems: 'center' },
-  content: { flexDirection: 'row', paddingHorizontal: 16 },
+  containerText: {paddingHorizontal: 16},
+  iconPlush: {fontSize: normalizeFontSize(24), color: COLORS.anon_primary},
+  views: {color: COLORS.anon_primary},
+  containerDetail: {flex: 1},
+  contentContainer: {flex: 1},
+  contentDetail: {flexDirection: 'row', alignItems: 'center'},
+  content: {flexDirection: 'row', paddingHorizontal: 16},
   descriptionContainer: {
     paddingHorizontal: 20,
     height: 0,
     flex: 1,
-    marginBottom: 14,
+    marginBottom: 14
   },
-  domainImage: { height: normalize(200), marginBottom: 14 },
-  domainImageEmptyState: { height: normalize(135), marginBottom: 14 },
+  domainImage: {height: normalize(200), marginBottom: 14},
+  domainImageEmptyState: {height: normalize(135), marginBottom: 14},
   wrapperItem: {
     backgroundColor: 'white',
     borderBottomWidth: 4,
-    borderBottomColor: COLORS.gray6,
+    borderBottomColor: COLORS.balance_gray,
     height: dimen.size.DOMAIN_CURRENT_HEIGHT - 2
   },
   wrapperImage: {
     borderRadius: normalize(45),
     borderWidth: 0.2,
-    borderColor: 'rgba(0,0,0,0.5)',
+    borderColor: COLORS.black50,
     width: normalize(48),
     height: normalize(48),
     justifyContent: 'center',
-    alignItems: 'center',
+    alignItems: 'center'
   },
   image: {
     height: normalize(48),
     width: normalize(48),
-    borderRadius: normalize(45),
+    borderRadius: normalize(45)
   },
   wrapperText: {
     backgroundColor: 'white',
     borderRadius: 8,
-    borderColor: '#00ADB5',
+    borderColor: COLORS.anon_primary,
     width: normalize(36),
     height: normalize(36),
     justifyContent: 'center',
     alignItems: 'center',
-    borderWidth: normalize(0.5),
+    borderWidth: normalize(0.5)
   },
   point: {
     width: 3,
     height: 3,
     borderRadius: 4,
-    backgroundColor: colors.gray,
+    backgroundColor: COLORS.blackgrey,
     marginLeft: 8,
-    marginRight: 8,
+    marginRight: 8
   },
-  height: (height) => ({ height }),
-  width: (width) => ({ width }),
+  height: (height) => ({height}),
+  width: (width) => ({width}),
   wrapperFooter: {
     paddingHorizontal: 8,
     height: normalize(52),
     // flexBasis: 52,
-    borderBottomColor: COLORS.gray1,
-    borderBottomWidth: 1,
+    borderBottomColor: COLORS.balance_gray,
+    borderBottomWidth: 1
   },
   headerDomainName: {
     fontSize: normalizeFontSize(14),
     fontFamily: fonts.inter[600],
     lineHeight: normalizeFontSize(16.9),
-    color: '#000000',
+    color: COLORS.black
   },
   headerDomainDate: {
     fontFamily: fonts.inter[400],
     fontSize: normalizeFontSize(12),
     lineHeight: normalizeFontSize(18),
-    color: '#828282',
+    color: COLORS.blackgrey
   },
   domainItemTitle: {
     fontSize: normalizeFontSize(16),
     fontFamily: fonts.inter[700],
-    lineHeight: normalizeFontSize(24),
+    lineHeight: normalizeFontSize(24)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color scheme has been updated in the StyleSheet. Ensure that these new colors align with the overall theme of the application and provide a consistent user experience. 



<!-- This is an auto-generated comment by OSS CodeRabbit -->